cypress/integration/Crosscutting/BackToTopTest/index.js

```diff
@@ -0,0 +1,18 @@
+/// <reference types="Cypress" />
+import { When, Then, And } from "cypress-cucumber-preprocessor/steps";
+
+When('user scrolls 600 px down', () => {
+    cy.scrollTo(0, 600);
+});
+
+Then('back-to-top arrow appears with text {string} and href {string}', (textTop, hrefTop) => {
+    cy.get('div.site-footer__container > a').should('have.text', textTop).and('have.attr', 'href', hrefTop);
```

Comment:
this is not checking for visibility - that is why your test for 600 px was passing where in fact it should not have

make it 
  cy.get('.back-to-top').should('be.visible').and('have.text', textTop).and('have.attr', 'href', hrefTop);
});

Reply:
done